src/open_inwoner/plans/managers.py

```diff
@@ -3,13 +3,4 @@
 
 class PlanQuerySet(QuerySet):
     def connected(self, user):
-        return self.filter(
-            Q(created_by=user)
-            | Q(contacts__contact_user=user)
-            | Q(contacts__created_by=user)
-        ).distinct()
-
-    def shared(self, user):
-        return self.filter(
-            Q(contacts__contact_user=user) | Q(contacts__created_by=user)
-        ).distinct()
+        return self.filter(Q(created_by=user) | Q(plan_contacts__id=user.id)).distinct()
```

Comment:
Maybe it makes sense to auto-add the creating user also to plan_contacts?

Reply:
Agree with Alex, it will simplify code in several places